packages/screens/RiotGame/component/CollectionView.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { ScrollView } from "react-native-gesture-handler";
+
+import { SortDirection } from "../../../api/marketplace/v1/marketplace";
+import { useCollectionInfo } from "../../../hooks/useCollectionInfo";
+import {
+  TabsListType,
+  Header as CollectionHeader,
+  Content as CollectionContent,
+} from "../../Marketplace/CollectionScreen";
```

Comment:
avoid importing across screens, better to move these components in components package

Reply:
Oki. I refactored those components and file ```Marketplace/CollectionScreen```